paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/instrumentation/ResourcesCompatVisitorFactory.kt

```diff
@@ -0,0 +1,18 @@
+package app.cash.paparazzi.gradle.instrumentation
+
+import com.android.build.api.instrumentation.AsmClassVisitorFactory
+import com.android.build.api.instrumentation.ClassContext
+import com.android.build.api.instrumentation.ClassData
+import com.android.build.api.instrumentation.InstrumentationParameters
+import org.objectweb.asm.ClassVisitor
+
+internal abstract class ResourcesCompatVisitorFactory : AsmClassVisitorFactory<InstrumentationParameters.None> {
+  override fun createClassVisitor(
+    classContext: ClassContext,
+    nextClassVisitor: ClassVisitor
+  ): ClassVisitor {
+    return ResourcesCompatTransform(instrumentationContext.apiVersion.get(), nextClassVisitor)
```

Comment:
question: why use the apiVersion vs Opcodes.ASM9?  I kinda like what you did here, but could upgrades to the underlying libs break us in the future?

Reply:
ya, I noticed that forcing the version caused issues. I saw this usage in the other pr and thought that might be a better way to go.

Looks like `Opcodes.ASM9` works